docs/quickstart/quickstart_chains/cosmos-agoric.md

```diff
@@ -0,0 +1,344 @@
+- [MessageHandlers](../../build/manifest/cosmos.md#mapping-handlers-and-filters): On each and every message that matches optional filter criteria, run a mapping function
+- [EventHanders](../../build/manifest/cosmos.md#mapping-handlers-and-filters): On each and every event that matches optional filter criteria, run a mapping function
+
+Note that the manifest file has already been set up correctly and doesn’t require significant changes, but you need to change the datasource handlers. This section lists the triggers that the manifest file looks for on the blockchain to start indexing.
+
+```ts
+{
+  dataSources: [
+    {
+      kind: CosmosDatasourceKind.Runtime,
+      startBlock: 11628269,
+      mapping: {
+        file: "./dist/index.js",
+        handlers: [
+          // {
+          //     Using block handlers slows your project down as they can be executed with each and every block.
+          //     Only use if you need to
+          //     handler: 'handleEvent',
+          //     kind: CosmosHandlerKind.Block,
+          // },
+          {
+            handler: "handleEvent",
+            kind: CosmosHandlerKind.Event,
+            filter: {
+              type: "transfer",
+              messageFilter: {
+                type: "/cosmos.bank.v1beta1.MsgSend",
+              },
+            },
+          },
+          {
+            handler: "handleMessage",
+            kind: CosmosHandlerKind.Message,
+            filter: {
+              type: "/cosmos.bank.v1beta1.MsgSend",
+            },
+          },
+        ],
+      },
+    },
+  ],
+}
+```
+
+In the code above we have defined two handlers. `handleEvent` will be executed whenever a `transfer` type is detected within a message filter of `/cosmos.bank.v1beta1.MsgSend` type. `handleMessage` is the other handler that will be triggered when a `/cosmos.bank.v1beta1.MsgSend` filter type is triggered. These handlers ared used to track the transfers and messages within the Agoric network.
+
+Check out our [Manifest File](../../build/manifest/cosmos.md) documentation to get more information about the Project Manifest (`project.ts`) file.
+
+## 2. Update Your GraphQL Schema File
+
+The `schema.graphql` file determines the shape of the data that you are using SubQuery to index, hence it's a great place to start. The shape of your data is defined in a GraphQL Schema file with various [GraphQL entities](../../build/graphql.md).
+
+For this project, you'll need to modify your schema.graphql file as follows. Since we're indexing all [reward transactions](https://www.mintscan.io/akash/txs/808FED7F3FE680EEF8E005EC1927C0CF00D2975E4B26CEE7A098D5DA7DEA8217?height=11797219) for delegators in the Akash network, we have a DelegatorReward entity that comprises a number of properties, including reward amount, delegator information, validator's address, and so forth.
```

Comment:
This is Akash?

Reply:
We're not indexing DelegatorReward entities

Reply:
Fixed.